### PR TITLE
Add German translation for start-folding.yaml

### PR DESCRIPTION
**Pull request by michael-raps**

### CODE REVIEW

File: Localization/de-DE/start-folding.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+# Third paragraph first sentence in bold
+#"Please be patient if you experience idle time as we face the COVID-19 pandemic together! ",
+"Bitte haben Sie etwas Geduld, falls Ihr Computer zwischendurch nichts zu tun hat, da wir alle von der COVID-19 Pandemie betroffen sind! ",
+
+# Rest of the third paragraph
+#"We are working like mad to start simulations of COVID-19 proteins. The enthusiastic response to our effort has been tremendous and has sometimes emptied our queues. This is a great “problem” to have as there is no end to the valuable science we can do, its just a matter of giving us some time to get more running.",
+"Wir arbeiten mit Hochdruck daran um mit Simulationen von COVID-19 Proteinen anzufangen. Die enthusiastischen Reaktionen auf unsere Bemühungen waren enorm und dies hat die Warteschlangen manchmal geleert. Das ist ein großartiges "Problem", denn die wertvollen Wissenschaft zu der wir fähig sind ist nicht abgeschlossen. Es geht bloß darum, uns etwas mehr Zeit zu geben, um mehr in Gang zu bringen.",
```
Comment (max-dau) [inline Localization/de-DE/start-folding.yaml:21]:
```suggestion
"Wir arbeiten mit Hochdruck daran, mit Simulationen von COVID-19 Proteinen anzufangen. Die enthusiastischen Reaktionen auf unsere Bemühungen waren enorm und dies hat die Warteschlangen manchmal geleert. Das ist ein großartiges "Problem", denn die wertvollen Wissenschaft zu der wir fähig sind ist nicht abgeschlossen. Es geht bloß darum, uns etwas mehr Zeit zu geben, um mehr in Gang zu bringen.",
```

File: Localization/de-DE/start-folding.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+"Die Folding@home Software läuft im Hintergrund.",
+
+
+# Second paragraph
+#"While you keep going with your everyday activities, your computer will be working to help us find cures for diseases like cancer, ALS, Parkinson’s, Huntington’s, Influenza and many others.",
+"Während Sie mit Ihren alltäglichen Dingen weitermachen, wird Ihr Computer uns dabei helfen Heilmittel für Krankheiten wie Krebs, ALS, Parkinson, Huntington, Grippe und viele andere zu finden.",
+
+# Third paragraph first sentence in bold
+#"Please be patient if you experience idle time as we face the COVID-19 pandemic together! ",
+"Bitte haben Sie etwas Geduld, falls Ihr Computer zwischendurch nichts zu tun hat, da wir alle von der COVID-19 Pandemie betroffen sind! ",
```
Comment (Tar-Minyatur) [inline Localization/de-DE/start-folding.yaml:17]: My gut feeling says that the `as` there is more meant like a `während`. The reasoning otherwise doesn't fit so well (i.e. why would the PC by idle because of COVID-19).

File: Localization/de-DE/start-folding.yaml (hunk 1 above)
Comment (Tar-Minyatur) [inline Localization/de-DE/start-folding.yaml:21]:
```suggestion
"Wir arbeiten mit Hochdruck daran um mit Simulationen von COVID-19 Proteinen anzufangen. Die enthusiastischen Reaktionen auf unsere Bemühungen waren enorm und dies hat die Warteschlangen manchmal geleert. Das ist ein großartiges „Problem”, denn die wertvollen Wissenschaft zu der wir fähig sind ist nicht abgeschlossen. Es geht bloß darum, uns etwas mehr Zeit zu geben, um mehr in Gang zu bringen.",
```

(See also the discussion we had in chat about quotation marks.)

File: Localization/de-DE/start-folding.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+#"We are working like mad to start simulations of COVID-19 proteins. The enthusiastic response to our effort has been tremendous and has sometimes emptied our queues. This is a great “problem” to have as there is no end to the valuable science we can do, its just a matter of giving us some time to get more running.",
+"Wir arbeiten mit Hochdruck daran um mit Simulationen von COVID-19 Proteinen anzufangen. Die enthusiastischen Reaktionen auf unsere Bemühungen waren enorm und dies hat die Warteschlangen manchmal geleert. Das ist ein großartiges "Problem", denn die wertvollen Wissenschaft zu der wir fähig sind ist nicht abgeschlossen. Es geht bloß darum, uns etwas mehr Zeit zu geben, um mehr in Gang zu bringen.",
+
+#
+# OS-dependent binary download box
+# TODO: needs translation as well!
+
+# Bold text
+#"Don’t see the correct software for your device? Try our list of",
+"Sie finden nicht die richtige Software für Ihr Gerät? Probieren Sie unsere Liste mit ",
```
Comment (Tar-Minyatur) [inline Localization/de-DE/start-folding.yaml:29]:
```suggestion
"Sie finden nicht die richtige Software für Ihr Gerät? Probieren Sie unsere Liste mit",
```

Not sure how important additional spaces are.

File: Localization/de-DE/start-folding.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+#"alternative downloads",
+"alternativen Downloads",
+
+# End of sentence punctuation
+#".",
+" aus.",
+
+# Bold text
+#"Additional platforms",
+"Weitere Platformen",
```
Comment (Tar-Minyatur) [inline Localization/de-DE/start-folding.yaml:41]:
```suggestion
"Weitere Plattformen",
```

File: Localization/de-DE/start-folding.yaml [hunk 5]
```diff
@@ -0,0 +1,47 @@
+---
+url: "https://foldingathome.org/"
+language: "de-de"
```
Comment (Tar-Minyatur) [inline Localization/de-DE/start-folding.yaml:3]:
```suggestion
language: "de-DE"
```
Has to be the same as in the folder name, I guess.

File: Localization/de-DE/start-folding.yaml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+title: "BEGINNE ZU FALTEN"
```
Comment (Tar-Minyatur) [inline Localization/de-DE/start-folding.yaml:4]: For the homepage I translated this as `JETZT MITFALTEN`, because it felt more natural and I wanted to avoid using `BEGINNEN SIE JETZT ZU FALTEN`, which we would have to use to follow the "formal language" decision. I don't mind either way, we just need to decide which version to go for to keep it consistent.

File: Localization/de-DE/start-folding.yaml (hunk 2 above)
Comment (michael-raps) [reply Localization/de-DE/start-folding.yaml:17]: If you read the sentence without the inserted subordinate clause it will be clear what is meant. But I will rephrase it to make it more clear. Pls wait for the update...

File: Localization/de-DE/start-folding.yaml [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+text: [
+# Bold first paragraph
+#"The Folding@home software runs while you do other things.",
+"Die Folding@home Software läuft im Hintergrund.",
+
+
+# Second paragraph
+#"While you keep going with your everyday activities, your computer will be working to help us find cures for diseases like cancer, ALS, Parkinson’s, Huntington’s, Influenza and many others.",
+"Während Sie mit Ihren alltäglichen Dingen weitermachen, wird Ihr Computer uns dabei helfen Heilmittel für Krankheiten wie Krebs, ALS, Parkinson, Huntington, Grippe und viele andere zu finden.",
```
[the remark this answers is not in the document]
Comment (michael-raps) [reply Localization/de-DE/start-folding.yaml:13]: Thank you for this. I will consider it.

File: Localization/de-DE/start-folding.yaml (hunk 1 above)
Comment (michael-raps) [reply Localization/de-DE/start-folding.yaml:21]: Thank you. I will consider the second suggestion und rephrase the first to improve it.

File: Localization/de-DE/start-folding.yaml (hunk 3 above)
Comment (michael-raps) [reply Localization/de-DE/start-folding.yaml:29]: The additional space is because the sentence is not finished there, it continuous on the next part. In the English original it is the same.

File: Localization/de-DE/start-folding.yaml (hunk 4 above)
Comment (michael-raps) [reply Localization/de-DE/start-folding.yaml:41]: Thanx

File: Localization/de-DE/start-folding.yaml (hunk 5 above)
Comment (michael-raps) [reply Localization/de-DE/start-folding.yaml:3]: Ok

File: Localization/de-DE/start-folding.yaml (hunk 6 above)
Comment (michael-raps) [reply Localization/de-DE/start-folding.yaml:4]: yeahh that´s better.

File: Localization/de-DE/start-folding.yaml (hunk 3 above)
Comment (Tar-Minyatur) [reply Localization/de-DE/start-folding.yaml:29]: The English original doesn't have a space there. That's why I pointed it out. :) Still not sure how important it is and it might not even matter at all as HTML ignores multiple spaces.

File: Localization/de-DE/start-folding.yaml [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+# Third paragraph first sentence in bold
+#"Please be patient if you experience idle time as we face the COVID-19 pandemic together! ",
+"Wir sind alle von der COVID-19 Pandemie betroffen, bitte haben Sie daher etwas Geduld, falls Ihr Computer zwischendurch nichts zu tun hat. ",
+
+# Rest of the third paragraph
+#"We are working like mad to start simulations of COVID-19 proteins. The enthusiastic response to our effort has been tremendous and has sometimes emptied our queues. This is a great “problem” to have as there is no end to the valuable science we can do, its just a matter of giving us some time to get more running.",
+"Wir arbeiten mit Hochdruck daran Simulationen von COVID-19 Proteinen zu ermöglichen. Die enthusiastischen Reaktionen auf unsere Bemühungen waren enorm und dies hat die Warteschlangen manchmal geleert. Das ist ein großartiges „Problem“, denn die wertvollen Wissenschaft zu der wir fähig sind ist nicht abgeschlossen. Es geht bloß darum, uns etwas mehr Zeit zu geben, um mehr in Gang zu bringen.",
```
Comment (Tar-Minyatur) [inline Localization/de-DE/start-folding.yaml:21]:
```suggestion
"Wir arbeiten mit Hochdruck daran Simulationen von COVID-19 Proteinen zu ermöglichen. Die enthusiastischen Reaktionen auf unsere Bemühungen waren enorm und dies hat die Warteschlangen manchmal geleert. Das ist ein großartiges „Problem“, denn die wertvolle Wissenschaft, zu der wir fähig sind, ist nicht abgeschlossen. Es geht lediglich darum, uns etwas mehr Zeit zu geben, um mehr in Gang zu bringen.",
```
One small typo, two missing commas (I think) and a suggestion for a slightly more formal phrase.

File: Localization/de-DE/start-folding.yaml [hunk 9]
```diff
@@ -10,15 +10,15 @@ text: [
 
 # Second paragraph
 #"While you keep going with your everyday activities, your computer will be working to help us find cures for diseases like cancer, ALS, Parkinson’s, Huntington’s, Influenza and many others.",
-"Während Sie mit Ihren alltäglichen Dingen weitermachen, wird Ihr Computer uns dabei helfen Heilmittel für Krankheiten wie Krebs, ALS, Parkinson, Huntington, Grippe und viele andere zu finden.",
+"Während Sie mit Ihren alltäglichen Dingen weitermachen, wird Ihr Computer uns dabei helfen Heilmittel für Krankheiten wie Krebs, ALS, Parkinson, Chorea Huntington, Grippe und viele andere zu finden.",
 
 # Third paragraph first sentence in bold
 #"Please be patient if you experience idle time as we face the COVID-19 pandemic together! ",
-"Bitte haben Sie etwas Geduld, falls Ihr Computer zwischendurch nichts zu tun hat, da wir alle von der COVID-19 Pandemie betroffen sind! ",
+"Wir sind alle von der COVID-19 Pandemie betroffen, bitte haben Sie daher etwas Geduld, falls Ihr Computer zwischendurch nichts zu tun hat. ",
```
Comment (max-dau) [inline Localization/de-DE/start-folding.yaml:17]:
```suggestion
"Wir sind alle von der COVID-19 Pandemie betroffen. Bitte haben Sie daher etwas Geduld, falls Ihr Computer zwischendurch nichts zu tun hat. ",
```

File: Localization/de-DE/start-folding.yaml [hunk 10]
```diff
@@ -10,15 +10,15 @@ text: [
...
 
 # Rest of the third paragraph
 #"We are working like mad to start simulations of COVID-19 proteins. The enthusiastic response to our effort has been tremendous and has sometimes emptied our queues. This is a great “problem” to have as there is no end to the valuable science we can do, its just a matter of giving us some time to get more running.",
-"Wir arbeiten mit Hochdruck daran um mit Simulationen von COVID-19 Proteinen anzufangen. Die enthusiastischen Reaktionen auf unsere Bemühungen waren enorm und dies hat die Warteschlangen manchmal geleert. Das ist ein großartiges "Problem", denn die wertvollen Wissenschaft zu der wir fähig sind ist nicht abgeschlossen. Es geht bloß darum, uns etwas mehr Zeit zu geben, um mehr in Gang zu bringen.",
+"Wir arbeiten mit Hochdruck daran Simulationen von COVID-19 Proteinen zu ermöglichen. Die enthusiastischen Reaktionen auf unsere Bemühungen waren enorm und dies hat die Warteschlangen manchmal geleert. Das ist ein großartiges „Problem“, denn die wertvollen Wissenschaft zu der wir fähig sind ist nicht abgeschlossen. Es geht bloß darum, uns etwas mehr Zeit zu geben, um mehr in Gang zu bringen.",
```
Comment (max-dau) [inline Localization/de-DE/start-folding.yaml:21]:
```suggestion
"Wir arbeiten mit Hochdruck daran Simulationen von COVID-19 Proteinen zu starten. Die enthusiastischen Reaktionen auf unsere Bemühungen waren enorm und dies hat die Warteschlangen manchmal geleert. Das ist ein großartiges „Problem“, denn die wertvollen Wissenschaft zu der wir fähig sind ist nicht abgeschlossen. Es geht bloß darum, uns etwas mehr Zeit zu geben, um mehr in Gang zu bringen.",
```

Comment (max-dau) [reply Localization/de-DE/start-folding.yaml:21]: just an option. sounds better in my opinion, but you choose.

File: Localization/de-DE/start-folding.yaml [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+title: "JETZT MITFALTEN"
+text: [
+# Bold first paragraph
+#"The Folding@home software runs while you do other things.",
+"Die Folding@home Software läuft im Hintergrund.",
+
+
+# Second paragraph
+#"While you keep going with your everyday activities, your computer will be working to help us find cures for diseases like cancer, ALS, Parkinson’s, Huntington’s, Influenza and many others.",
+"Während Sie mit Ihren alltäglichen Dingen weitermachen, wird Ihr Computer uns dabei helfen Heilmittel für Krankheiten wie Krebs, ALS, Parkinson, Chorea Huntington, Grippe und viele andere zu finden.",
```
[the remark this answers is not in the document]
Comment (max-dau) [reply Localization/de-DE/start-folding.yaml:13]: comma after "im Hintergrund dabei"?

[the remark this answers is not in the document]
Comment (dsdel) [reply Localization/de-DE/start-folding.yaml:13]: I think it is not necessary and is more fluent without

[the remark this answers is not in the document]
Comment (max-dau) [reply Localization/de-DE/start-folding.yaml:13]: ok

File: Localization/de-DE/start-folding.yaml (hunk 8 above)
Comment (michael-raps) [reply Localization/de-DE/start-folding.yaml:21]: thanx

File: Localization/de-DE/start-folding.yaml (hunk 9 above)
Comment (michael-raps) [reply Localization/de-DE/start-folding.yaml:17]: thanx again

File: Localization/de-DE/start-folding.yaml (hunk 10 above)
Comment (michael-raps) [reply Localization/de-DE/start-folding.yaml:21]: Yes, it´s nearer to the original.

File: Localization/de-DE/start-folding.yaml (hunk 11 above)
[the remark this answers is not in the document]
Comment (michael-raps) [reply Localization/de-DE/start-folding.yaml:13]: thanx

File: Localization/de-DE/start-folding.yaml [hunk 12]
```diff
@@ -10,15 +10,15 @@ text: [
 
 # Second paragraph
 #"While you keep going with your everyday activities, your computer will be working to help us find cures for diseases like cancer, ALS, Parkinson’s, Huntington’s, Influenza and many others.",
-"Während Sie mit Ihren alltäglichen Dingen weitermachen, wird Ihr Computer uns dabei helfen Heilmittel für Krankheiten wie Krebs, ALS, Parkinson, Chorea Huntington, Grippe und viele andere zu finden.",
+"Während Sie mit Ihren alltäglichen Dingen nachgehen, hilft uns Ihr Computer im Hintergrund dabei helfen Heilmittel für Krankheiten wie Krebs, ALS, Parkinson, Chorea Huntington, Grippe und viele andere zu finden.",
```
Comment (max-dau) [inline Localization/de-DE/start-folding.yaml:13]:
```suggestion
"Während Sie Ihren alltäglichen Dingen nachgehen, hilft uns Ihr Computer im Hintergrund dabei, Heilmittel für Krankheiten wie Krebs, ALS, Parkinson, Chorea Huntington, Grippe und viele andere zu finden.",
```

File: Localization/de-DE/start-folding.yaml [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+#"We are working like mad to start simulations of COVID-19 proteins. The enthusiastic response to our effort has been tremendous and has sometimes emptied our queues. This is a great “problem” to have as there is no end to the valuable science we can do, its just a matter of giving us some time to get more running.",
+"Wir arbeiten mit Hochdruck daran Simulationen von COVID-19 Proteinen zu starten. Die enthusiastischen Reaktionen auf unsere Bemühungen waren enorm und dies hat die Warteschlangen manchmal geleert. Das ist ein großartiges „Problem“, denn die wertvolle Wissenschaft, zu der wir fähig sind, ist nicht abgeschlossen. Es geht lediglich darum, uns etwas mehr Zeit zu geben, um mehr in Gang zu bringen.",
+
+#
+# OS-dependent binary download box
+# TODO: needs translation as well!
+
+# Bold text
+#"Don’t see the correct software for your device? Try our list of",
+"Sie finden nicht die richtige Software für Ihr Gerät? Probieren Sie unsere Liste mit ",
```
Comment (uyaem) [inline Localization/de-DE/start-folding.yaml:29]: It probably won't matter, but like with other files, it has been suggested to mirror trailing spaces (and likewise, not use trailing spaces when the original text doesn't have one.

File: Localization/de-DE/start-folding.yaml (hunk 12 above)
Comment (dsdel) [reply Localization/de-DE/start-folding.yaml:13]: ++ : )

Comment (michael-raps) [reply Localization/de-DE/start-folding.yaml:13]: uups...
